integration_tests/tests/test_logging/containerpilot.json5

```diff
@@ -0,0 +1,43 @@
+{
+  consul: "consul:8500",
+  logging: {
+    level: "DEBUG",
+    format: "text"
+  },
+  jobs: [
+    {
+      name: "job1",
+      // this echo gets wrapped in a log line
+      exec: "echo job1 exec",
+    },
+    {
+      name: "job2",
+      // this echo is not wrapped in a log line
+      exec: "echo job2 exec",
+      passthru: true,
```

Comment:
The only question I have about the name is if there's any other interaction besides stdout/logs that might get passed through. If so, the name might be ambiguous.

I'm wondering if the following might make more sense:

- `output: capture` (default, not needed in the config)
- `output: pas.*`

That might be easier to understand without referencing the docs, but it has the disadvantage of requiring a more sophisticated answer than true/false/1/0, etc.